src/Server/Repositories/ClaraJobsApi.cs

```diff
@@ -32,9 +32,11 @@ public class ClaraJobsApi : IJobs
         private readonly ILogger<ClaraJobsApi> _logger;
         private readonly IJobsClient _jobsClient;
 
-        public ClaraJobsApi(IOptions<DicomAdapterConfiguration> dicomAdapterConfiguration, ILogger<ClaraJobsApi> logger) : this(
```

Comment:
The client is actually provided by Platform in `Nvidia.Clara.Platform.IPayloadsClient`, to avoid confusion, this is placed inside the `Repositories` dir as an API.